test/blackbox-tests/test-cases/output-obj/dune

```diff
@@ -45,6 +53,13 @@
  (action
   (run ./static.bc)))
 
+;; This fails in travis.  Can't seem to find libcamlrun (see pull request #1229
+;; for discussion)
+(alias
```

Comment:
This needs to be commented so that Travis can succeed

Reply:
I merged master before pushing the commented test but I'm now getting a fail in:

./test/blackbox-tests/test-cases/tests-stanza/run.t

Reply:
`dune promote` should fix it. It puts the corrected run.t file in place